data/hai/hai reveal 1 intro.txt

```diff
@@ -11,48 +11,774 @@
+			`	She sighs deeply and says, "Naturally they've ramped up security, but half the Hai crew were still on board and we have no idea what their fate has been. It's got everyone a bit shaken and distrustful at the moment."`
+			`	"Oh, that's awful!" you say.`
+			`	She nods and forces a little bit of false cheer into her voice, "This is why we're all wearing pink, just a little show of local solidarity.`
+				goto main
+			label pink
+			`	A sad smile crosses her face and her expression goes somber as she explains, "A pirate marauder recently snuck down onto the planet and hijacked a ship straight off the landing pad. The Hai are in shock, it's just completely unprecedented for something like that to happen in broad daylight. Naturally, they've ramped up security, but half the Hai crew were still on board and we have no idea what their fate has been. Everyone is a bit shaken up so us locals are wearing pink in solidarity... and so we're not mistaken for a security risk.`
+			label main
+			`	"Unfortunately, it's taking a while for them to redistribute an appropriately resourced security force thanks to the Unfettered. So far they've only been able to recall those on leave and up their vigilance."`
+			label questions
+			choice
+				`	"Could I do anything to help?"`
+					goto help
+					to display
+						not "Hai Reveal Concern Period: Raid on Allhome (local): label help"
+				`	"Who are the Unfettered?"`
+					goto unfettered
+					to display
+						not "First Contact: Unfettered: offered"
+						not "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
+				`	"Thank you for your time."`
+					goto end
+					to display
+						or
+							has "Hai Reveal Concern Period: Raid on Allhome (local): label help"
+							has "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
+			label unfettered
+			action
+				set "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
+			`	She seems surprised that you asked and responds, "Oh, they're a sort of rebel Hai group. If you don't know you could go visit Cloudfire in the Wah Ki system up north, or maybe even Darkcloak if you dare. A short visit would explain it far better than I ever could.`
+			`	"Be very careful if you do though, they're very likely to shoot at you."`
+				goto questions
+			label help
+			action
+				set "Hai Reveal Concern Period: Raid on Allhome (local): label help"
+			`	She shrugs. "I've heard talk of some things that have been suggested to help, but I don't know myself. I guess you should just do the right thing, whatever that is."`
+			`	She seems to be at a loss for what else to say.`
+				goto questions
+			label end
+			`	"No, thank you for caring," she says. "Whatever you do, just be cautious about it. People are scared, and I have a hunch that when they're under pressure, the Hai are a little stranger than people think they are.`
+			`	"Safe travels, Captain."`
+				decline
+
```

Comment:
This will need to be done in the future for all of HR, but I don't think there's anything stopping us from doing this one right now.
```suggestion
	on decline
		clear "Hai Reveal Concern Period: Raid on Allhome (local): label help"
		clear "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
```